pkg/api/admin_provisioning.go

```diff
@@ -0,0 +1,30 @@
+package api
+
+import (
+	"context"
+	m "github.com/grafana/grafana/pkg/models"
+)
+
+func (server *HTTPServer) AdminProvisioningReloadDasboards(c *m.ReqContext) Response {
+	err := server.ProvisioningService.ProvisionDashboards()
```

Comment:
Injecting and calling the service here instead of using bus. Seems easier to follow the code that way to me but not sure we want to do it that way (part of my question for next backend weekly)

Reply:
Think this is aligned with how we want to build things moving forward. I like it :+1:

Reply:
Yea I like this. We want to get rid of the bus when possible. Thumbs up!